test/saithrift/tests/sail3.py

```diff
@@ -807,6 +807,241 @@ def runTest(self):
+
+            attr_value = sai_thrift_attribute_value_t(u32=hash_seed)
+            attr = sai_thrift_attribute_t(id=SAI_SWITCH_ATTR_ECMP_DEFAULT_HASH_SEED, value=attr_value)
+            self.client.sai_thrift_set_switch_attribute(attr)
+			
+	    for i in range(1, maximum_packets):
+                packet = simple_tcp_packet(eth_dst=router_mac,
+                                           eth_src='00:22:22:22:22:22',
+                                           ip_dst= ip_destination,
+                                           ip_src=ip_source,
+                                           tcp_sport=source_port,
+                                           tcp_dport=destination_port,
+                                           ip_id=106,
+                                           ip_ttl=64) 
+                #packet at port1 
+                expected_packet1 = simple_tcp_packet(eth_dst='00:11:22:33:44:51',   
+                                                     eth_src=router_mac,
+                                                     ip_dst= ip_destination,
+                                                     ip_src=ip_source,
+                                                     tcp_sport=source_port,
+                                                     tcp_dport=destination_port,
+                                                     ip_id=106,
+                                                     ip_ttl=63)  
+                #packet at port2 
+                expected_packet2 = simple_tcp_packet(eth_dst='00:11:22:33:44:52',
+                                                     eth_src = router_mac,     
+                                                     ip_dst= ip_destination,
+                                                     ip_src=ip_source,
+                                                     tcp_sport=source_port,
+                                                     tcp_dport=destination_port,   
+                                                     ip_id =106,       
+                                                     ip_ttl=63) 
+                #packet at port3  
+                expected_packet3 = simple_tcp_packet(eth_dst='00:11:22:33:44:53',
+                                                     eth_src = router_mac,     
+                                                     ip_dst= ip_destination,
+                                                     ip_src=ip_source,
+                                                     tcp_sport=source_port,
+                                                     tcp_dport=destination_port,     
+                                                     ip_id =106,       
+                                                     ip_ttl=63) 
+
+
+                send_packet(self, 3, str(packet)) 
+                rcv_idx = verify_any_packet_any_port(self, [expected_packet1, expected_packet2, expected_packet3], [0, 1, 2])
+                count[rcv_idx] += 1
+                source_port += 1
```

Comment:
Re-Initialize the src port and destination port to same values as given  for default hash seed , otherwise flow is  different  after changing hash seed value.

Reply:
Changes are made as per your comments.